### PR TITLE
174930703: Drop the WageVerification class and associated items

### PR DESCRIPTION
**Pull request by smedberg**
As discussed at https://usdrcovid19.slack.com/archives/C01874U5PDH/p1600788373017400

See https://www.pivotaltracker.com/n/projects/2459634/stories/174930703

### CODE REVIEW

**Review by MatthewSchultz — CHANGES_REQUESTED** [review]
Also remove from `routes.rb`?

**Comment by smedberg** [comment]
Good call on `routes.rb`- will remove now

**Comment by smedberg** [comment]
@MatthewSchultz Removed from `routes.rb`, and from a bunch of views, etc.  It kinda had it's hooks in a lot of places.  The app starts and seems to work to me, though I don't know how to test it thoroughly.

**Review by MatthewSchultz — CHANGES_REQUESTED** [review]
One quick change then is ready.

File: db/seeds.rb [hunk 1]
```diff
@@ -9,5 +9,5 @@
 Applicant.destroy_all
 Task.destroy_all
 
-FactoryBot.create_list(:applicant, 20, :with_wage_verification)
+FactoryBot.create_list(:applicant, 20)
```
[the remark this answers is not in the document]
Comment (smedberg) [reply db/seeds.rb:12]: @MatthewSchultz  Sorry, I don't think I understand what you're proposing.  If I revert the change to line 12, seeding doesn't work:
```
smedberg@MES-DESKTOP:~/code/papua-id-demo$ rake db:seed
rake aborted!
NameError: uninitialized constant WageVerification
```

My understanding is that adding `:with_wage_verification` means "fill in the WageVerification objects for these applicants."  But the purpose of this PR is to delete the `WageVerification` class, so I don't think that will work.

Am I misunderstanding?

[the remark this answers is not in the document]
Comment (MatthewSchultz) [reply db/seeds.rb:12]: you're right - if we are going to delete this class, we can't still have this.....

[the remark this answers is not in the document]
Comment (MatthewSchultz) [reply db/seeds.rb:12]: I wanted to make sure this didn't break `/admin/applicants/:id` - and it doesn't. PR approved as is.
image

**Review by MatthewSchultz — APPROVED** [review]

**Pull request merged**